README.md

```diff
@@ -102,6 +102,10 @@ The configuration sets up access to the devices via COAP protocol.
 | `apis.coap.inactivityMonitor.timeout` | string | `Time limit to close inactive connection.` | `20s` |
 | `apis.coap.blockwiseTransfer.enabled` | bool | `If true, enable blockwise transfer of coap messages.` | `false` |
 | `apis.coap.blockwiseTransfer.blockSize` | int | `Size of blockwise transfer block.` | `1024` |
+| `apis.coap.ownershipTransfer.method` | string | `Selected ownership transfer method. The supported values are: "justWorks", "manufacturer".` | `"justWorks"` |
```

Comment:
I would propose to use full name, as it's in the OCF. `Manufacturer Certificate` -> `manufacturerCertificate`

Reply:
Fixed